File: python-client-sdk/src/sdk/models/shared/codat_dataintegrity_contracts_metadata_matchconnectionids.py
from __future__ import annotations
import dataclasses
from dataclasses_json import Undefined, dataclass_json
from sdk import utils
from typing import Optional


@dataclass_json(undefined=Undefined.EXCLUDE)
@dataclasses.dataclass
class CodatDataIntegrityContractsMetadataMatchConnectionIds:
    source: Optional[list[str]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('source'), 'exclude': lambda f: f is None }})
    target: Optional[list[str]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('target'), 'exclude': lambda f: f is None }})
    

File: typescript-client-sdk/src/sdk/models/shared/codatdataintegritycontractsmetadatamatchconnectionids.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { Expose } from "class-transformer";


export class CodatDataIntegrityContractsMetadataMatchConnectionIds extends SpeakeasyBase {
  @SpeakeasyMetadata()
  @Expose({ name: "source" })
  source?: string[];

  @SpeakeasyMetadata()
  @Expose({ name: "target" })
  target?: string[];
}